DataFormats/Math/interface/PtEtaPhiMass.h

```diff
@@ -34,5 +34,24 @@ class PtEtaPhiMass {
 };
 
 
+class RhoEtaPhi {
```

Comment:
Because is not much different of the above, I would have then also to factorize in again another file etaFrom...

I personally find having data member first much more useful. I can make them public